src/main/java/org/jabref/JabRefException.java

```diff
@@ -3,11 +3,14 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+// Class for handling exceptions in GUI and printing corresponding exception messages
```

Comment:
Not a Javadoc class comment. No real added value by this comment: as this class is extending `Exception` it's purpose is obvious.

Reply:
Okay.
I understand this.